### PR TITLE
.gitattributes CrLf for JSONs

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
Running `regression-test.ps1` compares all the actual and expected files using `git diff`. This emits noisy useless warnings for each file.

This PR suppresses that to make the results easy to read.

>warning: CRLF will be replaced by LF in analyzers/its/./diff/expected/AutoMapper--net461-S1006.json.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in analyzers/its/./diff/actual/AutoMapper--net461-S1006.json.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in analyzers/its/./diff/expected/AutoMapper--net461-S103.json.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in analyzers/its/./diff/actual/AutoMapper--net461-S103.json.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in analyzers/its/./diff/expected/AutoMapper--net461-S1067.json.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in analyzers/its/./diff/actual/AutoMapper--net461-S1067.json.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in analyzers/its/./diff/expected/AutoMapper--net461-S107.json.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in analyzers/its/./diff/actual/AutoMapper--net461-S107.json.
The file will have its original line endings in your working directory

### CODE REVIEW

File: .gitattributes
Comment (andrei-epure-sonarsource) [inline .gitattributes:5]: I wonder, is there a possibility to create the files with LF, to avoid having LF for C#/VB.NET  and CRLF for json?

Comment (pavel-mikula-sonarsource) [reply .gitattributes]: That's a very good question

Comment (pavel-mikula-sonarsource) [reply .gitattributes]: Local repo will be still polluted with CRLF and the warning. To clear it:
* pull commits from this PR
* delete `expected/*.json`
* revert the deletion, git will recreate those local files with LF

I'll leave a message on our Slack about this once it's merged.

File: analyzers/its/expected/.gitattributes
Comment (pavel-mikula-sonarsource) [inline analyzers/its/expected/.gitattributes:2]: _Hint:_ The original `.gitignore` from the root was reverted. This is a change of another file inside `/expected` folder.

**Review by andrei-epure-sonarsource — APPROVED** [review]
LGTM!

I'll let you merge it in order to announce it on slack when you do it.

**Pull request merged**
